pom.xml

```diff
@@ -12,6 +12,8 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>1.7</maven.compiler.source>
+    	<maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
I used java8 to use all power of lambda expressions. We can downgrade to java7. It is ok.